logdna/utils.py

```diff
@@ -10,7 +10,15 @@ def is_jsonable(obj):
         return False
 
 
-def sanitize_meta(meta):
+def sanitize_meta(meta, index_meta=False):
+    if not index_meta:
+        if is_jsonable(meta):
+            return json.dumps(meta)
+
+        return {
+            '__errors': 'Meta cannot be serialized into JSON-formatted string'
+        }
```

Comment:
You may want to throw a try/except around `json.dumps`. If that fails for any reason, then it'll raise an exception that won't be handled here

Reply:
That's been tested within `is_jsonable` function; so, no need to add extra `try/except`